libs/ts/contracts/tasks/deployment-utils/init-chain.ts

```diff
@@ -1,101 +1,136 @@
-import type { Network, Signer } from 'ethers';
-import { Wallet, JsonRpcProvider } from 'ethers';
+import { Schema as S } from 'effect';
 
+import { type ethers as EthersType, id } from 'ethers';
+import { Wallet, JsonRpcProvider } from 'ethers';
```

Comment:
Why not keep the `import type {...}` and just update the line below with ethers functions and classes?

```suggestion
import type { ethers as EthersType } from 'ethers';
import { Wallet, JsonRpcProvider, id } from 'ethers';
```